doc/conf.py

```diff
@@ -105,16 +105,18 @@ def _warn_node(self, msg, node):
 
 # General information about the project.
 project = u'Luigi'
-copyright = u'2014, Erik Bernhardsson and Elias Freider'
+copyright = u'2015, Erik Bernhardsson and Elias Freider'
```

Comment:
I think this should be `2014-2015` - I think you need to include the year it is initially copyrighted. And with that I'm pretty sure it is actually earlier than 2014. git history looks like it should be 2011.